resources/tracing/README.md

```diff
@@ -1,7 +1,7 @@
-# Jaeger
+# Tracing
 
 ## Overview
-[Jaeger](http://jaeger.readthedocs.io/en/latest/) is a monitoring and tracing tool for microservice-based distributed systems.
+This component bundles [Jaeger](https://www.jaegertracing.io/) as a tracing tool for microservice-based distributed systems.
```

Comment:
```suggestion
This component bundles [Jaeger](https://www.jaegertracing.io/) which is a tracing tool for microservice-based distributed systems.
```
I'm not sure about `bundles` though - maybe `includes`? or rephrasing this sentence to `Tracing functionality is provided by...` something along these lines?

Reply:
Fixed by using third option